sf-stream/src/metrics.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{register_int_counter, IntCounter};
+use once_cell::sync::Lazy;
+
+pub static TRANSACTIONS_SENT: Lazy<IntCounter> = Lazy::new(|| {
+    register_int_counter!(
+        "aptos_sf_stream_transactions_sent_count",
+        "Transactions taken from the channel and pushed out to the SF server",
+    )
+    .unwrap()
+});
+
+pub static TRANSACTIONS_QUEUED: Lazy<IntCounter> = Lazy::new(|| {
+    register_int_counter!(
+        "aptos_sf_stream_transactions_queued_count",
+        "Transactions enqueued in the channel",
```

Comment:
Enqueued in what channel?

Should it say transactions queued to be sent to SF?

I also guess maybe processed or something, rather than queued.  Queued sounds active like it should be 5 if there are 5 in the channel right now.